Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -976,6 +983,48 @@ def get_incident_command(client: Client, args: dict[str, Any]) -> CommandResults
     return command_results
 
 
+def list_external_websites_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    list_external_websites command: Get external websites .
+
+    Args:
+        client (Client): CortexXpanse client to use.
+        args (dict): all command arguments, usually passed from ``demisto.args()``.
+            ``args['filter']`` Used for filter websites based on authentication type
+            ``args['limit']`` Used for limit num of results
+
+    Returns:
+        CommandResults: A ``CommandResults`` object that is then passed to ``return_results``
+    """
+    limit = int(args.get('limit', DEFAULT_SEARCH_LIMIT))
+    searchFilter = args.get('filter')
+    if limit > 500:
+        raise ValueError('Limit cannot be more than 500, please try again')
+
+    filters = {'filters': [], 'search_to': limit}
+    if searchFilter:
+        filters['filters'] = [{'field': 'authentication',
+                               'operator': 'contains',
+                               'value': searchFilter}]
+
+    response = client.get_external_websites(filters)
+
+    hosts = []
+    for each in response['reply']['websites']:
+        hosts.append({'Host': each['host'], 'Authentication type': each['authentication']})
+
+    human_readable = (f"Total results: {len(hosts)}\n \
+        {tableToMarkdown('External Websites', hosts, ['Host', 'Authentication type'])}" if hosts else "No Results")
+    command_results = CommandResults(
+        outputs_prefix='ASM.ExternalWebsite',
+        outputs_key_field='',
+        outputs=[response.get('reply')],
```

Comment:
```suggestion
        outputs=response.get('reply').get('websites')
```
This will make it if nothing comes back, nothing is added to context